Documentation/project-docs/platform-names.md

```diff
@@ -0,0 +1,39 @@
+# Platform Names
+
+Over the years, we've accumulated quite a few names, platforms, and components that can make it hard for folks to understand what we're referring to. This document has a list that will help to qualify what we mean by what.
+
+This will also list some aliases. As you'll see the aliases aren't always correct -- they are merely listed to help you find the better and less confusing terminology.
+
+## .NET Framework
+
+**Also referred to as**: Desktop, full framework, in-box framework
+
+This refers to the .NET Framework that shipped in 2002. It's the main framework folks target today and allows you to build a wide variety of applications, such as WinForms, WPF, ASP.NET, and command line tools.
+
+## .NET Core
```

Comment:
I can see that. The reason I call it out is because it's unfortunately quite visible. For instance, when you author a NuGet package that you want to consumer from Windows Store 8 apps, you've to create a folder `netcore45`. For Windows 10, it's `netcore50`. Also, the TFM is exposed in various places as well and reads `.NETCore, Version=4.5' for instance.

Thus I think including that context might is helpful with clarifying things.